### ISSUE TITLE
[BUG] Login to OCS Reports could take several minutes

### ISSUE THREAD

**Issue by Tarbos** [OCSInventory-NG/OCSInventory-ocsreports#1352]
**OCS Inventory version**
Version :  2.9.2 (only, versions up to 2.9.1 never showed this behavior)

**Describe the bug**
After pressing Send at the login page it can take up to four minutes till the next page shows up. The MySQL server is running at 100% cpu during that time and the mysql-slow.log generates an entry like this:

```
# Time: 220308 10:02:41
# User@Host: ocs[ocs] @ localhost []
# Thread_id: 203803  Schema: ocsweb  QC_hit: No
# Query_time: 71.275970  Lock_time: 0.000077  Rows_sent: 549  Rows_examined: 142606
# Rows_affected: 0
SET timestamp=1646730161;
SELECT DISTINCT n.netid as ipsubnet,s.name,s.id
            FROM netmap n LEFT JOIN subnet s ON s.netid=n.netid ,accountinfo a
                    WHERE (s.TAG IS NULL OR s.TAG = '') union select netid,name,id from subnet WHERE TAG IS NULL OR TAG = '';
```

**To Reproduce**
1. Go to 'login page'
2. Fill in credentials
3. Click on 'Send'
4. Wait for next page
5. Could take anything between a second and four minutes

**Expected behavior**
An immediate login like in previous OCS versions

**Comment by thenamelessthing** [comment]
On my side I resolved this problem by enabling these parameters.

![image](https://user-images.githubusercontent.com/10235448/158225841-47d52b0f-8668-4d6c-bc1a-47a26a590377.png)

This can take effect only after ocs cron job was run. (by default once a day I think)

**Comment by Mrtiv** [comment]
in my case this option was not working , cron job was not executing I correct it (from 10 minutes to 18 seconds) launchin in mysql purge query manually 
delete from netmap where date<"2019-12-31 23:59:59";

**Comment by charleneauger** [comment]
Hi @Tarbos ,

This bug has been fixed with the PR #1314.
To fix it, apply the fix manually or install a nightly version.

Best regards,
Charlene
